aspnetcore/fundamentals/apis.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "APIs overview"
+author: tdykstra
+description: Learn about differences between controller-based APIs and minimal APIs.
+ms.author: tdykstra
+ms.date: 01/10/2023
+uid: fundamentals/apis
+---
+
+# Choose between controller-based APIs and minimal APIs
+
+ASP.NET Core supports two approaches to creating APIs: a controller-based approach and minimal APIs. *Controllers* in an API project are classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. *Minimal APIs* are designed so that minimal code and configuration is required to create an API. This article points out differences between the two approaches.
```

Comment:
```suggestion
ASP.NET Core supports two approaches to creating APIs: a controller-based approach and minimal APIs. *Controllers* in an API project are classes that derive from <xref:Microsoft.AspNetCore.Mvc.ControllerBase>. *Minimal APIs* leverage endpoints defined with logical handlers in lambdas or method. This article points out differences between the two approaches.
```

Or some similar language. Essentially, I think we should highlight the distinction between class-based APIs and the more functional style that minimal APIs leverages.